sqlglot/parser.py

```diff
@@ -1032,6 +1034,7 @@ def _parse_create(self) -> t.Optional[exp.Expression]:
         elif create_token.token_type == TokenType.INDEX:
             this = self._parse_index()
         elif create_token.token_type in (
```

Comment:
Maybe we should factor this out into a class variable.

Reply:
52b32261